Assets/Scripts/PlayerController.cs

```diff
@@ -446,6 +470,21 @@ private void HandleGravity()
                 else accel = fallAccelerationDown;
 
                 _velocity.y = Mathf.MoveTowards(_velocity.y, -maxFallSpeed, accel * Time.fixedDeltaTime);
+                break;
+            case PlayerStateEnum.Down:
+                if (_velocity.y > -downSpeed)
+                {
+                    _velocity.y = -downSpeed;
+                }
+                else
+                {
+                    _velocity.y = math.max(_velocity.y * downAcceleration, -maxDownSpeed);
+                }
+                if (!Input.GetKeyDown(KeyCode.S)) //reverts to air if not holding 
```

Comment:
FYI `Input.GetKeyDown` only returns true the frame the player starts holding 'S', which means that as they continually hold S `Input.GetKeyDown` returns false. The only reason why you don't see an impact is because the frame rate is higher than the physics update rate, and `GetInput` in `Update` constantly corrects to the actual PlayerState.